commands/shortyShooter.py

```diff
@@ -41,7 +43,10 @@ def execute(self):
         else:
             self.shooter.runIntake(0)
 
-        self.shooter.runShooters(self.shooterSpeed())
+        voltage = 0
+        if self.shooterSpeed() > 0.2:
```

Comment:
```suggestion
        # set full speed if enabling shooter
        if self.shooterSpeed() > 0.2:
```

Reply:
Having conversations with yourself is healthy!